docs/core/whats-new/index.md

```diff
@@ -0,0 +1,160 @@
+
+- .NET Core console apps
+- .NET Core class libraries
+- .NET Standard class libraries
+- .NET Core unit test projects
+- .NET Core xUnit test projects 
+
+For example, to create a Visual Basic "Hello World" application, do the following steps from the command line:
+
+1. Open a console window, create a directory for your project, and make it the current directory.
+
+1. Enter the command `dotnet new console -lang vb`.
+
+   The command creates a project file with a `.vbproj` file extension, along with a Visual Basic source code file named *Program.vb*. This file contains the source code to write the string "Hello World!" to the console window.
+  
+1.  Enter the command `dotnet run`. The [.NET Core CLI tools](../tools/index.md) automatically compile and execute the application, which displays the message "Hello World!" in the console window.
+
+**Support for C# 7.1**
+
+.NET Core 2.0 supports C# 7.1, which adds a number of new features, including:
+
+- The `Main` method, the application entry point, can be marked with the [async](../../csharp/language-reference/keywords/async.md) keyword.
+- Inferred tuple names.
+- Default expressions.
+- Pattern matching with generics. 
+
+<!-- For more information see [link to C# what's new](url). -->
+
+## Platform improvements
+
+.NET Core 2.0 includes a number of features that make it easier to install .NET Core and to use it on supported operating systems.
+
+### .NET Core for Linux is a single implementation**
+
+.NET Core 2.0 offers a single Linux implementation that works on multiple Linux distributions. .NET Core 1.x required that you download a distribution-specific Linux implementation.
+
+You can also develop apps that target Linux as a single operating system. .NET Core 1.x required that you target each Linux distribution separately. 
+
+### Support for the Apple cryptographic libraries
+
+.NET Core 1.x on macOS required the OpenSSL toolkit's cryptographic library. .NET Core 2.0 uses the Apple cryptographic libraries and doesn't require OpenSSL, so you no longer need to install it. 
+
+## API changes and library support
+
+### Support for .NET Standard 2.0
+
+The .NET Standard defines a versioned set of APIs that must be available on .NET implementations that comply with that version of the standard. The .NET Standard is targeted at library developers. It aims to guarantee the functionality that is available to a library that targets a version of the .NET Standard on each .NET implementation. .NET Core 1.x supports the .NET Standard version 1.6; .NET Core 2.0 supports the latest version, .NET Standard 2.0.  For more information, see [.NET Standard](../../standard/net-standard.md).
+
+The .NET Standard 2.0 includes over 20,000 more APIs than were available in the .NET Standard 1.6. Much of this expanded surface area results from incorporating APIs that are common to the .NET Framework and Xamarin into .NET Standard.
```

Comment:
I recommend removing "the" from before ".NET Standard" when talking about a specific version of it